fastn/src/main.rs

```diff
@@ -61,6 +53,10 @@ async fn fastn_core_commands(matches: &clap::ArgMatches) -> fastn_core::Result<(
     use colored::Colorize;
     use fastn_core::utils::ValueOf;
 
+    if matches.subcommand_name().is_none() {
```

Comment:
This change was required as call to `fastn_core_commands()` calls `fastn_core::Config::read()` which errors when used outside of package dir even when no subcommand used in the cli.